### DISCUSSION TITLE
Profile to just recover deleted files

### DISCUSSION THREAD

**Discussion by andrefranciscopjr** [sepinf-inc/IPED#2109] [General]
Boa noite pessoal, espero que todos estejam  bem, haveria alguma forma de de criar perfil de recuperração de dados do n iped para excutar somente esta tarefa  , fotos , documentos, imagens, emails, e entre outros, somente para agilizar na procura  tentei pegar da pasta conf e e criar um txt  com com o nome IPEDConfig  e coloquei as opcçoes de neste txt e criei uma pasta conf  e coloquei as opções porem  na hora de rodar a iso nao  deu certo se pudeem me ajudar fico agradecido.

**Comment by lfcnassif** [comment]
Hi. You can create a profile disabling all options inside IPEDConfig.txt, but keep `enableCarving` and `processFileSignatures` enabled. The last is needed by the first to detect if a recovered zip is actually a docx, xlsx or pptx, if a recovered ole container is a doc, xls or ppt, if a recovered QuickTime container is qt, mov, mp4 or 3gp, for example. You also need to enable `addUnallocated` inside conf/FileSystemConfig.txt

**Reply by wladimirleite** [reply]
Just adding some basic information about profiles, you should create the "IPEDConfig.txt" @lfcnassif mentioned in a folder like this:
`profiles\recover\IPEDConfig.txt`
In this example, "recover" is the profile name.
And when running IPED, use `-profile recover` in the command line.
Look at other profiles in "profiles" folder for examples and at [this Wiki section](https://github.com/sepinf-inc/IPED/wiki/User-Manual#-profile).

**Reply by andrefranciscopjr** [reply]
Bom dia muito obrigado  vou fazer esse procedimentos muito obrigado @lfcnassif  e @wladimirleite , qualquer duvida pegunto novamente um otimo domingo
